### ISSUE TITLE
Error: ZodBigInt is not supported by zu.coerce

### ISSUE THREAD

**Issue by jpgilchrist** [JacobWeisenburger/zod_utilz#9]
I saw your posts through out zod issues and discussions and decided to give zod_utilz a try. Unfortunately, I encountered this error

```
Error: ZodBigInt is not supported by zu.coerce
```

```ts
z.object({
  id: id: zu.coerce(z.bigint()),
  ...
})
```

I have encountered many zod resolution issues through different packages and have zod resolved at 3.22.4.

**Comment by jpgilchrist** [comment]
That being said, I did clone your rep and run the tests and the test does pass:

[```test( 'coerce( z.bigint() )', () => {`](https://github.com/JacobWeisenburger/zod_utilz/blob/12aa7fd0d5482f54606bb6a0426d909089fb23a2/src/coerce.test.ts#L56)

**Comment by JacobWeisenburger** [comment]
I think this should be fixed in: [0.8.0](https://github.com/JacobWeisenburger/zod_utilz/releases/tag/0.8.0)

Would you be willing to check?
